Source/Core/Runtime/Configuration/RuntimeConfigurator.cs

```diff
@@ -187,9 +190,8 @@ public static RuntimeConfigurator Instance
             {
                 if (Exists == false)
                 {
-                    throw new NullReferenceException("Process runtime configurator is not set in the scene. Create an empty game object with the 'RuntimeConfigurator' script attached to it.");
+                    Debug.LogWarning("Process runtime configurator is not set in the scene. Create an empty game object with the 'RuntimeConfigurator' script attached to it.");
```

Comment:
Actually I wouldn't want to change the "Runtime configuration missing" errors to warnings. I would argue they are legitimate errors.
I'm ok with the warning that "there is no registry so the object will not be registered", but is there a way to avoid this? It might be possible to check RuntimeConfigurator.Exists in the Process Scene Object instead of calling the getter/setter and getting here.